examples/components/icons/icons.jsx

```diff
@@ -3,7 +3,7 @@
 const React = require('react');
 
 const Icon = require('../../../src').Icon;
-require('../../design/icon/icon.styl');
```

Comment:
I didn't get why - but bem-loader tries to load design/icon/icon.styl file and couldn't do this